client/src/modules/user/model/actions.js

```diff
@@ -138,3 +148,14 @@ export const changePassword = (password, newPassword, newPasswordConfirm) =>
     newPassword,
     newPasswordConfirm
   });
+
+export const updateSettings = settings => dispatch =>
+  postData('/auth/update-settings', { settings })
```

Comment:
Consider just `POST /auth/settings`. Please discuss with the team change of the `auth` route to `user` or `profile` as this is no longer `auth` only endpoint.

Reply:
We will discuss this.